db.go

```diff
@@ -40,7 +40,7 @@ func (p Postgres) MigrationLogDeleteSql() string {
 type Mysql struct{}
 
 func (m Mysql) SelectMigrationTableSql() string {
-	return "SELECT table_name FROM information_schema.tables WHERE table_name = ?"
+	return "SELECT table_name FROM information_schema.tables WHERE table_name = ? AND table_schema = ?"
```

Comment:
I see. We would be giving the user the ability to run a migration on any database the server has access to? The only reason I'm concerned about this is because the behavior for MySQL isn't exactly the same as for Postgres.